tests/testUtils.py

```diff
@@ -244,18 +245,18 @@ def writeCSV(self, data, fileName):
 
     {'id1' : {'f1':n1, 'f2':n2}, 'id2' : {'f1':n3, 'f2':n4}}
     """
-    csvFile = open(fileName, 'wb')
+    csvFile = open(fileName, 'w')
     csvFileWriter = csv.writer(csvFile)
     # get the headers from the first row
-    header = sorted(data[data.keys()[0]].keys())
+    header = list(data[list(data.keys())[0]].keys())
```

Comment:
Makes sense. Glad I asked :smile: 

Here is a PR you could further improve. See https://github.com/Radiomics/pyradiomics/pull/197